addons/web/static/src/views/form/form_controller.scss

```diff
@@ -1117,7 +1118,13 @@
 
 .o_form_view.o_xxs_form_view {
     .o_inner_group {
-        @include form-break-table;
+        .o_wrap_field {
+            .o_cell {
+                .o_field_widget {
+                    margin-bottom: 0;
```

Comment:
As a general approach, we do tend to customise these properties contextually and by using CSS variables.

Depending by the amount of changes necessary, you may want to take the approach in consideration.
I'm not saying that you have to, a one-line fix would be preferred indeed, I'm just exposing some possibilities 😉

More info:
https://www.odoo.com/documentation/master/contributing/development/coding_guidelines.html#css-variables